vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -28146,7 +28215,7 @@ unit:MilliDARCY
 unit:MilliDEG_C
   a qudt:DerivedUnit ;
   a qudt:Unit ;
-  dcterms:description "$Millidegree Celsius is a scaled unit of measurement for temperature.$"^^qudt:LatexString ;
+  dcterms:description "$\\textit{Millidegree Celsius}$ is a scaled unit of measurement for temperature."^^qudt:LatexString ;
```

Comment:
Ralph, is this one OK? Didn't you say something about starting a string with $ ?